src/test/java/org/sagebionetworks/web/unitclient/widget/entity/EntityMetadataTest.java

```diff
@@ -78,8 +84,9 @@
 	@Before
 	public void before() {
 		MockitoAnnotations.initMocks(this);
-		widget = new EntityMetadata(mockView, mockDoiWidget, mockAnnotationsWidget,
-				mockFileHistoryWidget, mockJsClient, mockJSNI, mockRestrictionWidgetV2);
+		widget = new EntityMetadata(mockView, mockDoiWidget, mockDoiWidgetV2, mockAnnotationsWidget,
+				mockFileHistoryWidget, mockJsClient, mockJSNI, mockRestrictionWidgetV2, mockCookies);
+		when(mockCookies.getCookie(any(String.class))).thenReturn(null);
```

Comment:
add test for alpha mode (configure new doi widget).  we can remove old test on release